app/screens/phone-auth-screen/phone-registration-validation.tsx

```diff
@@ -189,7 +188,6 @@ export const PhoneRegistrationValidateScreen: React.FC<
         logAddPhoneAttempt()
         const { data } = await phoneValidate({
           variables: { input: { phone, code } },
-          refetchQueries: [AccountScreenDocument],
```

Comment:
why is that no longer needed?

Reply:
Caching. Relevant fields can be queried back in the mutation and everywhere those fields get used will be updated automatically.